### DISCUSSION TITLE
Test case failing for cli.spec.ts

### DISCUSSION THREAD

**Discussion by dthadi3** [verdaccio/verdaccio#2106] [Q&A]
One test case is failing for node version (15.x)
FAIL  test/unit/modules/cli/cli.spec.ts
Is there any resolution for that?
FYI my travis build
https://travis-ci.com/github/dthadi3/verdaccio/builds/218245894

**Comment by juanpicado** [comment]
hi @dthadi3 , 🤔 well I rely on GH Actions since quite a while. I'm not sure which branch you are running but here are pretty green. The master branch is quite different from the branch 4.x, I hope that improve your experience with Travis.
